src/utils.rs

```diff
@@ -38,6 +38,21 @@ pub fn exec_cmd(cmd: &str, args: &[&str]) -> Option<CommandOutput> {
         0 => String::from(cmd),
         _ => format!("{} {}", cmd, args.join(" ")),
     };
+
```

Comment:
Did you mean to include these changes in this PR?

Reply:
Doh, no I was going to open a different PR for the Java changes. I'll pop this commit out.